src/VS.Mefx/Commands/WhiteList.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace VS.Mefx.Commands
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Text.RegularExpressions;
+
+    internal class WhiteList
+    {
+        // Constants associated with the Regex's for the expressions specified in the whitelist
+        private static readonly TimeSpan MaxRegexTime = new TimeSpan(0, 0, 5);
```

Comment:
`new TimeSpan(0, 0, 5)` leaves it difficult for the reader to know what units are used. I had to look it up.
Prefer syntax like `TimeSpan.FromSeconds(5)`.